Sources/SmileID/Classes/SmileID.swift

```diff
@@ -90,13 +112,70 @@ public class SmileID {
         }
     }
 
-    public class func cleanup(jobIds: String...) throws {
-        try cleanup(jobIds: jobIds)
-    }
+    /// Submits a previously captured job to SmileID for processing.
+    ///
+    ///- Parameters:
+    ///   - jobId: The unique identifier for the job to be submitted.
+    public class func submitJob(
+        jobId: String,
+        deleteFilesOnSuccess: Bool
+    ) throws {
+        let jobIds = LocalStorage.getSubmittedJobs()
+        if !jobIds.contains(jobId) {
+            throw SmileIDError.invalidJobId
+        }
 
-    /// deletes the job ids list, whether in unsubmitted or submitted state
-    public class func cleanup(jobIds: [String]) throws {
-        try LocalStorage.delete(at: jobIds)
+        Task {
+            let zip: Data
+            do {
+                let authRequestFile = try? LocalStorage.fetchAuthenticationRequestFile(jobId: jobId)
+                let authRequest = AuthenticationRequest(
+                    jobType: authRequestFile!.jobType,
```

Comment:
Instead of doing an force unwrap of `authRequestFile`, we should use the `guard-else-throw` pattern